calcom/cal.com
# test: add google calendar e2e

- chore: add google calendar E2E variables
- test: implement base google calendar tests

## What does this PR do?



Fixes #9371 

/claim #9371

## Type of change




## How should this be tested?



- Are there environment variables that should be set?
  - It is needed to set `E2E_TEST_GOOGLE_CALENDAR_EMAIL` and `E2E_TEST_GOOGLE_CALENDAR_PASSWORD`
